pom.xml

```diff
@@ -2896,7 +2896,7 @@
               <compilerPlugin>
                 <groupId>com.github.ghik</groupId>
                 <artifactId>silencer-plugin_${scala.version}</artifactId>
-                <version>1.7.10</version>
+                <version>1.7.13</version>
```

Comment:
Do we still need this plugin?
<img width="844" alt="image" src="https://github.com/apache/spark/assets/5399861/ff7cf96e-2d7f-4f1f-a1e8-74d9d8558a43">
https://github.com/ghik/silencer

Reply:
I think we still need this, `@nowarn` needs to identify the suppressed code one by one, which may lead to many unnecessary code changes

Reply:
OK.